eq_translations/cli/translate_census.py

```diff
@@ -40,20 +40,16 @@ def main():
     file_prefix = "individual" if "individual" in args.SCHEMA_PATH else "household"
     template_file = f"census_{file_prefix}.pot"
     output_file = f"census_{file_prefix}-cy.po"
+    project_api_key = os.getenv("CROWDIN_PROJECT_API_KEY")
 
```

Comment:
```suggestion
```

The new line indicates this var is significantly different to the prior vars, though it isn't initially obvious why.